src/librustc_span/symbol.rs

```diff
@@ -408,6 +408,7 @@ symbols! {
         label_break_value,
         lang,
         lang_items,
+        lazy_normalization_consts,
```

Comment:
(Do we really want a separate feature for this?)

Reply:
We already use const generics in libcore/libstd, so it might be better to not use lazy normalization there for now. I can try and restrict these changes on `const_generics` instead.

Reply:
We do want two features: lazy normalisation unblocks some issues even without const generics. We probably do always want lazy normalisation enabled when const generics is, though. (But I think this is the current effect anyway?)

Reply:
> (But I think this is the current effect anyway?)

No, rn lazy normalization is checked separately from const generics, so we need to enable both to use it.
Considering that we still eagerly normalize array lengths, I don't actually know how to use this on stable.